src/lib/mina_base/account_update.ml

```diff
@@ -457,7 +458,7 @@ module Update = struct
         ~checked:(js_only (Js_layout.leaf_type (Custom "TokenSymbol")))
         ~name:"TokenSymbol" string
     in
-    finish "PartyUpdate" ~t_toplevel_annots
+    finish "AccountUpdateUpdate" ~t_toplevel_annots
```

Comment:
😬

Reply:
Can we rename this? I think the OCaml `Account_update.Update.t` is fine as is, but this one is just too nasty.

I'd suggest `AccountUpdateChange`?

Reply:
I'm going with `AccountUpdateModification`